packages/tpetra/core/src/Tpetra_Details_Behavior.hpp

```diff
@@ -200,6 +200,15 @@ class Behavior {
   /// separate question.
   static size_t longRowMinNumEntries ();
 
+  /// \brief the threshold for transitioning from device to host
+  ///
+  /// If the number of elements in the multivector exceeds this 
```

Comment:
Should be "doesn't exceed" rather than "exceeds," right?

Reply:
@vbrunini Can you please fix this comment? Then I'll approve it immediately and you can set AUTOMERGE.

Reply:
I don't think I can push to @GeoffDanielson 's PR.

Reply:
@vbrunini Sorry, I thought this was your PR